src/EditorFeatures/Core/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -107,18 +107,26 @@ internal static class GoToDefinitionHelpers
             IStreamingFindUsagesPresenter streamingPresenter,
             CancellationToken cancellationToken,
             bool thirdPartyNavigationAllowed = true)
+        {
+            return threadingContext.JoinableTaskFactory.Run(
```

Comment:
what else is still using this helper?  The QI navigation and roslyn VS workspace bits are both gone, does that just leave the reference in AbstractGoToDefinitionService - https://sourceroslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/GoToDefinition/GoToDefinitionHelpers.cs,f200b1f10122b8c0,references

Can that be removed / or this method made obsolete?

Reply:
that's next step.  however that interface is used by TS (we don't have EA layer for it yet).